notebook/agenteval_cq_math.ipynb

```diff
@@ -0,0 +1,785 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "-pftZ-ZF1_BA"
+      },
+      "source": [
+        "<a href=\"https://colab.research.google.com/drive/1G4nqqcp-MZvxO75mVKuOm6TNIQ7eTmuw?usp=sharing\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "NPUGFpKP1_BH"
+      },
+      "source": [
+        "# Demonstrating the `AgentEval` framework using the task of solving math problems as an example\n",
+        "\n",
+        "This notebook aims to demonstrate how to `AgentEval` implemented through [AutoGen](https://github.com/microsoft/autogen) works, where we use a math problem-solving task as an example. \n",
+        "`AgentEval` consists of two key components:\n",
+        "\n",
+        "- `CriticAgent`: This is an LLM-based agent that generates a list criteria $(c_1, \\dots, c_n)$ to help to evaluate a utility given task.\n",
+        "\n",
+        "- `QuantifierAgent`: This agent quantifies the performance of any sample task based on the criteria designed by the `CriticAgent` in the following way: $(c_1=a_1, \\dots, c_n=a_n)$\n",
+        "\n",
+        "![AgentEval](../website/blog/2023-11-11-AgentEval/img/agenteval-CQ.png)\n",
+        "\n",
+        "For more detailed explanations, please refer to the accompanying [blog post](https://https://microsoft.github.io/autogen/blog/2023/11/11/AgentEval)\n",
+        "\n",
+        "## Requirements\n",
+        "\n",
+        "AutoGen requires `Python>=3.8`. To run this notebook example, please install pyautogen, Docker, and OpenAI:\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "execution": {
+          "iopub.execute_input": "2023-02-13T23:40:52.317406Z",
+          "iopub.status.busy": "2023-02-13T23:40:52.316561Z",
+          "iopub.status.idle": "2023-02-13T23:40:52.321193Z",
+          "shell.execute_reply": "2023-02-13T23:40:52.320628Z"
+        },
+        "id": "68lTZZyJ1_BI",
+        "outputId": "15a55fab-e13a-4654-b8cb-ae117478d6d8"
+      },
+      "outputs": [],
+      "source": [
+        "%pip install pyautogen~=0.2.0b5 docker\n",
+        "%pip install openai\n",
```

Comment:
it'll be installed by pyautogen automatically.
```suggestion
```

Reply:
great!